experiments/build-swiftshader.md

```diff
@@ -0,0 +1,22 @@
+# Building SwiftShader with Coverage Enabled
+
+The SwiftShader CMakeLists.txt file defines the SWIFTSHADER\_EMIT\_COVERAGE flag to generate coverage information. However, it was designed for a very specific toolchain (clang-10 toolchain, CMake option SWIFTSHADER\_LLVM\_VERSION=10.0).
+
+The build at https://swiftshader-review.googlesource.com/c/SwiftShader/+/44728 can be built with coverage information without these restrictions.
```

Comment:
I suggest removing this. The CL has been merged. You can now just clone SwiftShader and build it.